src/service/alerts/alert_manager.rs

```diff
@@ -17,55 +17,21 @@ use std::str::FromStr;
 
 use chrono::{Duration, FixedOffset, Utc};
 use config::{
-    cluster::LOCAL_NODE_UUID,
     meta::{
         stream::StreamType,
         usage::{TriggerData, TriggerDataStatus, TriggerDataType},
     },
     CONFIG,
 };
 use cron::Schedule;
-use infra::{dist_lock, scheduler};
+use infra::scheduler;
 
 use crate::{
-    common::{
-        infra::cluster::get_node_by_uuid,
-        meta::{alerts::AlertFrequencyType, dashboards::reports::ReportFrequencyType},
-    },
+    common::meta::{alerts::AlertFrequencyType, dashboards::reports::ReportFrequencyType},
     service::{db, usage::publish_triggers_usage},
 };
 
 pub async fn run() -> Result<(), anyhow::Error> {
-    // maybe in the future we can support multiple organizations
-    let org_id = "default";
-    // get the working node for the organization
-    let node = db::alerts::alert_manager::get_mark(org_id).await;
-    if !node.is_empty() && LOCAL_NODE_UUID.ne(&node) && get_node_by_uuid(&node).await.is_some() {
-        log::debug!("[ALERT_MANAGER] is processing by {node}");
-        return Ok(());
-    }
-
-    // before start merging, set current node to lock the organization
-    let lock_key = format!("/alert_manager/organization/{org_id}");
-    let locker = dist_lock::lock(&lock_key, 0).await?;
-    // check the working node for the organization again, maybe other node locked it
-    // first
-    let node = db::alerts::alert_manager::get_mark(org_id).await;
-    if !node.is_empty() && LOCAL_NODE_UUID.ne(&node) && get_node_by_uuid(&node).await.is_some() {
-        log::debug!("[ALERT_MANAGER] is processing by {node}");
-        dist_lock::unlock(&locker).await?;
-        return Ok(());
-    }
-    let ret = if node.is_empty() || LOCAL_NODE_UUID.ne(&node) {
-        db::alerts::alert_manager::set_mark(org_id, Some(&LOCAL_NODE_UUID.clone())).await
-    } else {
-        Ok(())
-    };
-    // already bind to this node, we can unlock now
-    dist_lock::unlock(&locker).await?;
-    drop(locker);
-    ret?;
```

Comment:
Due to the row update logic changes from @hengfeiyang ? ( Trying to understand the logic behind removal )

Reply:
@ansrivas @Subhra264 also used select for update to handle multiple alertManger, so don't need this part.